ui-manchette/src/components/OperationalPoint.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { OperationalPointType } from '../types/pathPropertiesTypes';
+import '@osrd-project/ui-core/dist/theme.css';
+import { positionKm } from './helpers';
+
+const OperationalPoint: React.FC<OperationalPointType> = ({ extensions, id, part, position }) => {
```

Comment:
part is not used, and actually you only need the following props: name, id, position and ch
```suggestion
const OperationalPoint: React.FC<OperationalPointType> = ({ id, position, name, ch }) => {
```